core/pipeline/label.go

```diff
@@ -82,19 +82,24 @@ func LoadGitLabels(workdir string) ([]Label, error) {
 		return nil, err
 	}
 
+	labels := []Label{}
+
 	origin, err := repo.Remote("origin")
-	if err != nil {
-		return nil, err
-	}
+	if err == nil {
+		urls := origin.Config().URLs
```

Comment:
Should be mutually exclusive; this was erroring out because the test I added doesn't set up an `origin`, so it must have been getting some sort of OriginNotFound error.